src/Codeception/Subscriber/Console.php

```diff
@@ -536,9 +557,17 @@ public function printScenarioTrace(ScenarioDriven $failedTest)
                 $message->style('bold');
             }
 
-            $line = $step->getLine();
-            if ($line and (!$step instanceof Comment)) {
-                $message->append(" at <info>$line</info>");
+            if (!$step instanceof Comment) {
+                $filePath = $step->getFilePath();
+                if ($filePath) {
+                    // Clickable `editorUrl`:
+                    if (isset($this->options['editorUrl']) and is_string($this->options['editorUrl'])) {
+                        $lineString = str_replace(['%%file%%', '%%line%%'], [codecept_absolute_path($step->getFilePath()), $step->getLineNumber()], $this->options['editorUrl']);
+                    } else {
+                        $lineString = $step->getFilePath() . ':' . $step->getLineNumber();
+                    }
+                }
+                $message->append(" at <info>$lineString</info>");
```

Comment:
if `$filePath` is null, $lineString is undefined, please move this line inside `if`.

Reply:
:-)